src/components/Content/Popup/Popup.js

```diff
@@ -84,7 +84,7 @@ Popup.propTypes = {
 	/** Whether popup should be made visible. Default: true */
 	show: PropTypes.bool,
 	/** Target node used for positioning of popup relative to it. Popper.js lib used behind the scene. */
-	targetNode: PropTypes.node,
+	targetNode: PropTypes.any,
```

Comment:
I already addressed this in my PBI (merged to master already). It's more specific and relies on Poper.js declaration. Please, rebase